client/keys/export_test.go

```diff
@@ -53,7 +53,7 @@ func Test_runExportCmd(t *testing.T) {
 			extraArgs:      []string{"--unsafe", "--unarmored-hex"},
 			userInput:      "y\n",
 			mustFail:       false,
-			expectedOutput: "2485e33678db4175dc0ecef2d6e1fc493d4a0d7f7ce83324b6ed70afe77f3485\n",
+			expectedOutput: "22667038337039573670342f526531793456773576493968437730544273794e4b6733755355626d672b68673d22\n",
```

Comment:
assuming this is due to the change from Secp256k1 -> Sr25519

Reply:
why change from 32 bytes key to longer bytes key? It's not suitable with any wallet.